lesson14.md

```diff
@@ -0,0 +1,178 @@
+<!-- .slide: id="lesson14" -->
+
+# Basic Frontend - Spring 2021
+
+Lesson 14, Tuesday, 2021-04-20
+
+---
+
+### Recap: DOM
+
+**DOM** lets us manipulate HTML elements from JavaScript. It's a very powerful **API**.
+
+---
+
+
+
+---
+
+<!-- How do we add a new property "foo" to an object?
+
+a) x += { foo: 42 };
+b) x += foo: 42;
+c) x.foo = 42;
+d) x = x + foo: 42;
+
+API stands for
+
+a) Abstract Particle Injector
+b) Application Programming Interface
+c) A Potential Interface
+d) All Possible Injectors
+
+DOM stands for
+
+a) Document Object Model
+b) Documented Object Model
+c) Dumb Object Mess
+d) Deliberate Opinionated Mode
+
+How do we get an element by Id?
+
+* document.giveMeElement("id")
+* document.elementById("id")
+* document.getElementById("id")
+* document.GetElementByIdPlease("id")
+
+--->
+
+---
+
+HTML:
+```html
+<div id="myDiv">Hello</div>
+<input id="myInput" type="number" value="42" />
+<button id="myButton">Click me!</button>
+```
+
+Select the elements from JavaScript:
+```js
+let myDivElement = document.getElementById("myDiv");s
+let myInputElement = document.getElementById("myInput");
+let myButtonElement = document.getElementById("myButton");
+```
+
+---
+
+```js
+let myDivElement = document.getElementById("myDiv");s
```

Comment:
there's an "s" too much

Reply:
Haha yeah, my command key gets stuck sometimes. So now you know I tried to save the file at that point ;)